### ISSUE TITLE
Filter by fixed combination of ingredients

### ISSUE THREAD

**Issue by catalamarti** [darwin-eu/CodelistGenerator#58]
Christian's suggestion: use dose form table and return those with the two (or more) pre-specified ingredients
Christian will give the sql @cgreich

**Comment by edward-burn** [comment]
@catalamarti is this different to https://github.com/darwin-eu/CodelistGenerator/issues/57?

**Comment by catalamarti** [comment]
Yes, the idea of this one is that you say some ingredients and you obtain the concepts that contain this ingredients.
